traffic_ops/traffic_ops_golang/server/servers_assignment.go

```diff
@@ -269,7 +277,9 @@ INSERT INTO deliveryservice_server (deliveryservice, server)
 
 	//need remap config location
 	var atsConfigLocation string
-	if err := tx.QueryRow("SELECT value FROM parameter WHERE name = 'location' AND config_file = '" + atscfg.RemapFile + "'").Scan(&atsConfigLocation); err != nil {
+	if err := tx.QueryRow("SELECT value FROM parameter " +
+		"WHERE name = 'location'" +
+		" AND config_file = '" + RemapFile + "'").Scan(&atsConfigLocation); err != nil {
```

Comment:
Done, goland did this multiline concatenation for me and that really isn't nitpicking it will highlight the backtick delimited string appropriately unlike the previous string

Reply:
Goland can't highlight that properly? I expected more from them.

Reply:
I think it's a style guide thing I missed that they adhere to? Probably could be configured but this is a mostly stock installation